apis/datadoghq/v1alpha1/datadogagentprofile_types.go

```diff
@@ -34,7 +34,12 @@ type Config struct {
 }
 
 type Override struct {
+	// Configure the basic configurations for an Agent container
+	// Valid Agent container names are: `agent`
 	Containers map[commonv1.AgentContainerName]*Container `json:"containers,omitempty"`
+
+	// If specified, indicates the pod's priority
```

Comment:
do you think we need more info in the docstring similar to [here](https://github.com/DataDog/datadog-operator/blob/main/apis/datadoghq/v2alpha1/datadogagent_types.go#L1022)?